src/main/java/org/spongepowered/api/world/explosion/Explosion.java

```diff
@@ -161,4 +232,4 @@ static Builder builder() {
         Explosion build() throws IllegalArgumentException;
 
     }
-}
+}
```

Comment:
Add new line at end of file, and then squash the commits, many thanks!

Reply:
How do I squash the commits?

Reply:
And I tried quickly adding a new line but I think I messed up. :(

Reply:
@gabizou If this is ready to merge otherwise, GitHub can squash the commits when the PR is merged.